README.md

```diff
@@ -31,12 +32,13 @@ cd forklift
 vagrant up centos7-foreman-nightly
 ```
 
-The same can be quickly done for a development environment:
+The same can be quickly done for a development environment where GITHUB_NICK is your GitHub username:
 
 ```
 git clone https://github.com/katello/forklift.git
 cd forklift
 cp boxes.yaml.example boxes.yaml
+sed -i "s/<REPLACE ME>/GITHUB_NICK/g" boxes.yaml
```

Comment:
This was a missing piece to the dev quickstart that should make this work out directly for developers.